torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1399,8 +1453,14 @@ class InsertQuantDeQuantHelper {
   // Record qscheme for every graph, this is for checking
   // each graph is only quantized with one type of QScheme
   std::unordered_map<Graph*, c10::QScheme> qscheme_for_graph_;
+
+  bool is_dynamic = false;
 };
 
+void InsertQuantDeQuantHelper::setDynamicFlag(bool is_dynamic_) {
```

Comment:
nit: you can move this to where it is declared since this is simple

Reply:
I think you want to rename the member `is_dynamic` to `is_dynamic_` to indicate that it is a member variable